.github/actions/build_portDNN_action/action.yml

```diff
@@ -17,11 +17,12 @@ runs:
     - name: Set up Environment and build portDNN
       shell: bash
       run: |
-        ls
-        export LD_LIBRARY_PATH=${{ inputs.workspace }}/linux_nightly_release/lib:$LD_LIBRARY_PATH
-        export CMAKE_CXX_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang++
-        export CMAKE_C_COMPILER=${{ inputs.workspace }}/linux_nightly_release/bin/clang
+        export LD_LIBRARY_PATH=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/lib/libsycl.so:${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/lib:$LD_LIBRARY_PATH
+        export CMAKE_CXX_COMPILER=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/bin/compiler/clang++
+        export CMAKE_C_COMPILER=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/bin/compiler/clang
         export OCL_ICD_FILENAMES=${{ inputs.workspace }}/install/lib/libCL.so
+        export DPCPP_INCLUDE_DIR=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/include/sycl
+        export PATH=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/include/sycl:${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/bin:${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/lib:$PATH
 
-        cmake -B portDNN_build_dir portDNN -GNinja -DCMAKE_CXX_COMPILER=$(pwd)/linux_nightly_release/bin/clang++ -DSNN_BUILD_BENCHMARKS=OFF -DSNN_BENCH_SYCLBLAS=OFF -DSNN_BUILD_DOCUMENTATION=OFF
+        cmake -B portDNN_build_dir portDNN -GNinja -DCMAKE_CXX_COMPILER=${{ inputs.workspace }}/intel/oneapi/compiler/2024.0/bin/compiler/clang++ -DSNN_BUILD_BENCHMARKS=OFF -DSNN_BENCH_SYCLBLAS=OFF -DSNN_BUILD_DOCUMENTATION=OFF
```

Comment:
You're exporting `CMAKE_CXX_COMPILER` above to the same value as you're setting `-DCMAKE_CXX_COMPILER` here. Could you just have, say, `-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}`? That's not getting into the question of whether both are actually required to be set or not.